### PR TITLE
newlogd and cgroups fixes

### PR DESCRIPTION
**Pull request by giggsoff**
Several modifications related with newlogd
1. We should set cgroup limits for all services and for newlogd as well
2. We should slightly expand ctrd_mem limit as we put user containerd there
3. We fill devSourceBytes map in newlogd with various sources and should cleanup it periodically
4. Unbounded read into memory may hit problem, let limit reader with reasonable size and use
line-by-line approach in newlogd gzip
5. To not consume memory for the content of the directory we should read
several files and iterate over them, not over the whole directory

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -375,8 +375,8 @@ Example:
 ```text
 set_global hv_dom0_mem_settings "dom0_mem=800M,max:800M"
 set_global hv_dom0_cpu_settings "dom0_max_vcpus=1 dom0_vcpus_pin"
-set_global hv_eve_mem_settings "eve_mem=500M,max:500M"
+set_global hv_eve_mem_settings "eve_mem=6500M,max:6500M"
```
Comment (eriknordmark) [inline docs/README.md:378]: Did you intentionally want to have this example be 13x larger or did you intend to add 150M as for the example below?

Comment (giggsoff) [reply docs/README.md]: Oh, typo, fixed

File: pkg/newlog/cmd/newlogd.go [hunk 2]
```diff
@@ -1019,18 +1020,22 @@ func doMoveCompressFile(tmplogfileInfo fileChanInfo) {
 	outfile := gzipFileNameGet(isApp, timenowNum, dirName, appuuid, tmplogfileInfo.notUpload)
 
 	// read input file
-	ifile, err := os.Open(tmplogfileInfo.tmpfile)
+	// with double overhead for limit
+	content, err := utils.ReadWithMaxSize(log, tmplogfileInfo.tmpfile, int(2*maxLogFileSize))
```
Comment (eriknordmark) [inline pkg/newlog/cmd/newlogd.go:1024]: Can't we do better by creating a reader and feeding the reader into a gzip function operating on a reader?

**Comment by giggsoff** [comment]
> Also, do the eden tests exercise the newlogd/loguploader gzip logic?

Yes, and they are failing with my changes as I can see (app_logs test), I'll try to figure out what's going on.

**Comment by giggsoff** [comment]
Still has no ideas why we miss DomainStatus and why we are so lucky in CI to hit it in 4 of 4 attempts and only one time from my side when I mentioned it in comment here.
Removed Dockerfile and dependencies updates for now.
Added another commit to not consume memory for the whole content of the directory.

**Review by eriknordmark — APPROVED** [review]
Run eden again

**Pull request merged**